webapp/src/ts/services/hydrate-contact-names.service.ts

```diff
@@ -35,7 +35,7 @@ export class HydrateContactNamesService {
 
   private getMutedState(summaries, contactSummaries) {
     summaries.forEach((summary) => {
-      if (summary.muted || !summary.lineage || !summary.lineage.length) {
+      if (!summary || summary.muted || !summary.lineage || !summary.lineage.length) {
         return;
```

Comment:
Both of these fixes were required, as lots of tests were generating feedback docs because of this missed nullchecks.